statsmodels/stats/moment_helpers.py

```diff
@@ -87,67 +109,88 @@ def mnc2cum(mnc):
 
     http://en.wikipedia.org/wiki/Cumulant#Cumulants_and_moments
     '''
-    mnc = [1] + list(mnc)
-    kappa = [1]
-    for nn,m in enumerate(mnc[1:]):
-        n = nn+1
-        kappa.append(m)
```

Comment:
it might be enough to use a copy of m here, e.g. by making mnc into a copy of the argument mnc at the beginning of the function.
question is whether we want to support list of lists unchanged or convert to numpy asarray if mnc is not 1-dimensional.